.scripts/create_push_client.sh

```diff
@@ -13,6 +14,7 @@ function git_configure_and_clone() {
 function generate_client_and_create_pr() {
     cwd=$(pwd)
     echo $cwd
+    make clean-generated
```

Comment:
Wouldn't it clean too much? I mean would it clean only the stuff generated for the client?

Reply:
addressed in ae6a925